sdk/src/cameras/ad-96tof1-ebz/camera_96tof1.cpp

```diff
@@ -340,10 +336,118 @@ aditof::Status Camera96Tof1::getDetails(aditof::CameraDetails &details) const {
     return status;
 }
 
-std::shared_ptr<aditof::CameraSpecifics> Camera96Tof1::getSpecifics() {
-    return m_specifics;
-}
-
 std::shared_ptr<aditof::DeviceInterface> Camera96Tof1::getDevice() {
     return m_device;
 }
+
+aditof::Status Camera96Tof1::getAvailableControls(std::vector<std::string> &controls) const {
+    using namespace aditof;
+    Status status = Status::OK;
+
+    controls = m_availableControls;
+
+    return status;
+}
+
+aditof::Status Camera96Tof1::setControl(const std::string &control, const std::string &value) {
+    using namespace aditof;
+    Status status = Status::OK;
+
+    std::vector<std::string>::iterator it;
+    it = std::find(m_availableControls.begin(), m_availableControls.end(), control);
+    if (it != m_availableControls.end()) {
```

Comment:
Shouldn't this be `it == m_availableControls.end()`. I'm reading this if like this, if `control` is not found in `m_availableControls` then it means this camera doesn't support that control.

I would also write `auto it = std::find...` as it is pretty clear for me that it is an iterator. Just my personal preference, can be ignored.

Reply:
Done